Add sitemap section to website strategy (#856)
solid/solidproject.org: website-strategy.md

1. Enterprises: https://solidproject.org/implement with aim to convince enterprises to implement Solid
2. Developers: https://solidproject.org/developers with aim to start building an app
3. Users: https://solidproject.org/users/get-a-pod with aim to start using an app

## Sitemap

> The sitemap information is intended to help track URIs that are allocated to the resources under solidproject.org. The information *is* incomplete (as of 2024-03-05) and needs to be updated as content and URI strategy changes (see also https://github.com/solid/solidproject.org/issues/249 ). The table should take 2xx, 3xx, 4xx status codes into account for all assets, including documents and media files that are made available as well as that were available in the past ( redirect, not found, gone..).

Reviewer: ```suggestion
The sitemap information is intended to help track URIs that are allocated to the resources under [`solidproject.org`](https://solidproject.org/). The information *is incomplete* (as of 2024-03-05) and will need to be updated as content and URI strategy changes (see also https://github.com/solid/solidproject.org/issues/249). The table should take 2xx, 3xx, 4xx status codes into account for all assets, including documents and media files that are newly made available, as well as those that were available in the past (redirect, not found, gone, etc.).
```

> * url: URLs of the tracked resources
> * status: intended status code in response to `HTTP GET`
> * design: information about the design (including architectural, presentational, behavioural)  of the resource
> * note: supplementary notes

Reviewer: ```suggestion
* url: URL of the tracked resources
* status: intended status code in response to `HTTP GET`
* design: information about the design (including architectural, presentational, behavioural) of the resource
* note: supplementary notes

The table is alphabetically sorted by url.
```

| https://solidproject.org/TR/2022/websocket-subscription-2021-20220509 | 200 | Uses W3C specification design patterns | - |
| https://solidproject.org/TR/acp | 200 | Uses W3C specification design patterns | - |
| https://solidproject.org/TR/notification-subscription-types | 200 | Uses W3C specification design patterns | - |
| https://solidproject.org/TR/notifications-protocol | 200 | Uses W3C specification design patterns | - |
| https://solidproject.org/TR/oidc | 200 | Uses W3C specification design patterns | - |
| https://solidproject.org/TR/oidc-primer | 200 | Uses W3C specification design patterns | - |
| https://solidproject.org/TR/protocol | 200 | Uses W3C specification design patterns | - |
| https://solidproject.org/TR/protocol.timemap | 200 | Uses W3C specification design patterns | - |
| https://solidproject.org/TR/wac | 200 | Uses W3C specification design patterns | - |
| https://solidproject.org/TR/wac.timemap | 200 | User-agent default. HTML+RDFa representation available | - |
| https://solidproject.org/TR/websocket-subscription-2021 | 200 | Uses W3C specification design patterns | - |
| https://solidproject.org/users/get-a-pod | | | |

> The table is alphabetically sorted by url.

Reviewer: ```suggestion
```

1. Enterprises: https://solidproject.org/implement with aim to convince enterprises to implement Solid
2. Developers: https://solidproject.org/developers with aim to start building an app
3. Users: https://solidproject.org/users/get-a-pod with aim to start using an app

## Sitemap

> The sitemap information is intended to help track URIs that are allocated to the resources under [`solidproject.org`](https://solidproject.org/). The information *is incomplete* (as of 2024-03-05) and will need to be updated as content and URI strategy changes (see also https://github.com/solid/solidproject.org/issues/249). The table should take 2xx, 3xx, 4xx status codes into account for all assets, including documents and media files that are newly made available, as well as those that were available in the past (redirect, not found, gone, etc.).

Reviewer: ```suggestion
The sitemap is intended to help track URIs that are allocated to the resources under [`solidproject.org`](https://solidproject.org/). The sitemap data *is incomplete* (as of 2024-03-05) and will need to be updated as content and URI strategy change (see also https://github.com/solid/solidproject.org/issues/249). The table should take 2xx, 3xx, 4xx status codes into account for all assets, including documents and media files that are newly made available, as well as those that were available in the past (redirect, not found, gone, etc.).
```

* url: URL of the tracked resource
* status: intended status code in response to `HTTP GET`
* design: information about the design (including architectural, presentational, behavioural) of the resource
* note: supplementary notes

> The table is alphabetically sorted by url.

Reviewer: ```suggestion
The table is alphabetically sorted by lexical url.
```

## Sitemap

The sitemap information is intended to help track URIs that are allocated to the resources under [`solidproject.org`](https://solidproject.org/). The information *is incomplete* (as of 2024-03-05) and will need to be updated as content and URI strategy changes (see also https://github.com/solid/solidproject.org/issues/249). The table should take 2xx, 3xx, 4xx status codes into account for all assets, including documents and media files that are newly made available, as well as those that were available in the past (redirect, not found, gone, etc.).

> * url: URL of the tracked resources

Reviewer: whoops! missed this one...
```suggestion
* url: URL of the tracked resource
```
